docs/audit_report/Makefile

```diff
@@ -3,7 +3,7 @@
 
 # You can set these variables from the command line, and also
 # from the environment for the first two.
-SPHINXOPTS    ?=
+SPHINXOPTS    ?= -W --keep-going
```

Comment:
We currently also explicitly set this options in the `url_check` job. Unsure if we want to remove it there now.

Reply:
Nah, I think that's fine. Let them all fail if something's wrong.